### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kannanlakshmi**
Updated quick start section to include a link to CockroachCloud

### CODE REVIEW

**Review by lnhsingh — COMMENTED** [review]
Small change, otherwise LGTM. You might need an eng review since this is in the `cockroach` repo, but not sure since this is only a README update

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/62448#-:-MWUHl1E1xlD7ru0ZoTS:bdxdgc5)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @kannanlakshmi)

___
*[README.md, line 62 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/62448#-MWUHZ3B31Igf1gs_kaP:-MWUHZ3B31Igf1gs_kaQ:b-nrnj0) ([raw file](https://github.com/cockroachdb/cockroach/blob/309623edd00c2ed35cee1cf34098f452920f536c/README.md#L62)):*
> ```Markdown
> - [Test Deployments](https://www.cockroachlabs.com/docs/stable/deploy-a-test-cluster.html) - Easiest way to test an insecure, multi-node CockroachDB cluster.
> - Production Deployments
>     - [CockroachCloud] (https://www.cockroachlabs.com/docs/cockroachcloud/quickstart) - Steps to deploy a free CockroachCloud cluster on public Cloud platforms.
> ```

`[CockroachCloud] (https://...)` should be `[CockroachCloud](https://...)` (no space between the two)

File: README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ CockroachDB supports the PostgreSQL wire protocol, so you can use any available
 
 - [Test Deployments](https://www.cockroachlabs.com/docs/stable/deploy-a-test-cluster.html) - Easiest way to test an insecure, multi-node CockroachDB cluster.
```
Comment (jseldess) [inline README.md:60]: Let's remove this Test Deployments link. The page it links to is gone, though we have a redirect in place.

**Review by j-low — APPROVED** [review]

**Pull request merged**